tests/Unit/Numerical/Spectral/Test_DefiniteIntegral.cpp

```diff
@@ -13,14 +13,20 @@
 #include "Numerical/Spectral/LegendreGaussLobatto.hpp"
 #include "tests/Unit/TestHelpers.hpp"
 
+// GCC 4.7.3 triggers the following for the CHECK() macros.
```

Comment:
which compiler triggers the warning?

Reply:
`GCC 5 triggers the -Wparentheses warning for some CHECK() macros, sometimes...`